jvm-libs/metrics/micrometer/src/main/kotlin/net/consensys/linea/metrics/micrometer/MicrometerHistogramAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package net.consensys.linea.metrics.micrometer
+
+import io.micrometer.core.instrument.DistributionSummary
+import net.consensys.linea.metrics.Histogram
+
+class MicrometerHistogramAdapter(private val adapter: DistributionSummary) : Histogram {
```

Comment:
`record` is the only useful method here. Linea code shouldn't care about the rest, because it's taken care of by Micrometer. Look at the `Counter` for example. It has only methods to write data, not to read it. Because reading it is not a responsibility of the code we write